net/acme-common/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=acme-common
-PKG_VERSION:=1.4.2
+PKG_VERSION:=1.4.5
```

Comment:
No need to bump the package version three times in a single PR

Reply:
Ok I'll do it . I will then make a separate commit which then increments PKG_RELEASE by one.